pkgs/servers/dns/knot-resolver/default.nix

```diff
@@ -38,6 +39,11 @@ unwrapped = stdenv.mkDerivation rec {
     # ExecStart can't be overwritten in overrides.
     # We need that to use wrapped executable and correct config file.
     sed '/^ExecStart=/d' -i systemd/kresd@.service.in
+  ''
+    # some tests have issues with network sandboxing, apparently
+  + optionalString doInstallCheck ''
+    sed '1i os.exit(77)' -i daemon/lua/trust_anchors.test/bootstrap.test.lua
```

Comment:
Maybe?

```
echo 'os.exit(77)' > daemon/lua/trust_anchors.test/bootstrap.test.lua
```

Reply:
Well, why not 🤷🏽   Similar enough, same end result.  It doesn't seem important to me at all.

Reply:
It is just slightly more readable.